atest/testdata/standard_libraries/string/string.robot

```diff
@@ -86,3 +86,27 @@ Get Substring With Invalid Start
 Get Substring With Invalid End
     [Documentation]    FAIL ValueError: Cannot convert 'end' argument 'invalid' to an integer.
     Get Substring    Hello Robot    2    invalid
+
+Strip String
+    ${result} =    Strip String  ${SPACE}${SPACE}Hello${SPACE}
```

Comment:
Four spaces between keywords and arguments would be a good. I know existing tests don't use that style consistently, but we are trying to move to that direction.